tests/functional/contribute/test_friends.py

```diff
@@ -13,3 +14,29 @@ def test_display_signup_form(base_url, selenium):
     page = ContributeFriendsPage(base_url, selenium).open()
     page.click_show_signup_form()
     assert page.is_signup_form_displayed
+    assert '' == page.email
+    assert 'United States' == page.country
+    assert page.html_format_selected
```

Comment:
Nit: as we're testing more than opening the form now, suggest renaming this test to `test_signup_default_values`?